docs/api/js/accounts-l1-l2.md

```diff
@@ -191,25 +192,18 @@ async requestExecute(transaction: {
-                                                                                                        |
-| overrides (optional)               | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice` etc.
-                                |
-| transaction.gasPerPubdataByte (optional)      |
-                              | 
-| transaction.refundRecipient(optional)         |  
-|--                                                                                                                                                         
-| returns                               | `PriorityOpResponse` object.                                                                                                                                                                                                                                                                                                                                     
+| Name                                     | Description                                                                                                                                     |
+| ---------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| transaction.contractAddress              | The L2 receiver address.                                                                                                                         |
+| transaction.calldata                     | The input of the L2 transaction.                                                                                                                 |
+| transaction.l2GasLimit                   | Maximum amount of L2 gas that transaction can consume during execution on L2.                                                                    |
+| transaction.l2Value?           | `msg.value` of L2 transaction.                                                                                                                   |
+| transaction.factoryDeps?       | An array of L2 bytecodes that will be marked as known on L2.                                                                                     |
+| transaction.operatorTip?       | If the ETH value passed with the transaction is not explicitly stated in the overrides, this field will be equal to the tip the operator will receive on top of the base cost of the transaction. |                                                                                                                                         
```

Comment:
Please introduce a note (similar to the one that was here before) that this field has not meaning as of now

Reply:
@StanislavBreadless  I have a chat with @vladbochok  and he told me that the ` This value has no meaning for the `Deque` type of queue, but it will be used to prioritize the transactions that get into the `Heap` or `HeapBuffer` queues. ` is outdated now

Reply:
@AnastasiiaVashchuk it is outdated because `Deque`/`Heap` are no longer present (we only have `Deque` rn). But the value is still unused, so please at least say that it is not needed right now.